code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -318,17 +318,29 @@
 		for(var/obj/item/organ/I in H.internal_organs)
 			I.status |= ORGAN_ADV_ROBOT
 
-/datum/species/proc/hug(var/mob/living/carbon/human/H,var/mob/living/target)
```

Comment:
I am not a fan of removing hug mechanics. Please add it as either *emote or something else. But it should still remain in game.

Reply:
![image](https://user-images.githubusercontent.com/17379794/53211874-d974b280-3653-11e9-95bf-812556f20ac7.png)

Reply:
Well it's fine if you make default help click as tap, but please keep the hug proc in and make it as emote or something else for people who actually want to hug. Plus it has a sound for it.

Reply:
But people can already hug themselves via me "", and there ALSO seems to be an existing emote? Why exactly should the hugging mechanics be preserved? Removing them, surprisingly enough, does not disallow you to hug.